dashboard/app/controllers/datasets_controller.rb

```diff
@@ -8,6 +10,15 @@ def show_manifest
   end
 
   def updated_manifest
+    parsed_manifest = JSON.parse(params['manifest'])
+    response = @firebase.set_library_manifest parsed_manifest
+    if response.success?
+      render json: {status: response.code}
+    else
+      render json: {status: response.code, msg: response.body}
+    end
```

Comment:
super nit / very subjective thought on simplifying this code -- feel free to ignore:

```ruby
my_json = {status: response.code}
my_json[:msg] = response.body unless response.success?

render json: my_json
```